RecursiveExtractor/FileEntry.cs

```diff
@@ -161,7 +190,7 @@ public FileEntry(string name, Stream inputStream, FileEntry? parent = null, bool
         /// <param name="content">The Stream to parse</param>
         /// <param name="parent">The Parent FileEntry</param>
         /// <returns>A FileEntry object holding a Copy of the Stream</returns>
-        public static async Task<FileEntry> FromStreamAsync(string name, Stream content, FileEntry? parent = null)
+        public static async Task<FileEntry> FromStreamAsync(string name, Stream content, FileEntry? parent = null, DateTime? createTime = null, DateTime? modifyTime = null, DateTime? accessTime = null, int memoryStreamCutoff = 1024*1024)
```

Comment:
Is this too small? Thought I saw 1 GB above somewhere.

Reply:
I'm using 1GB in tests.  The default so far has been 1MB, but I'm not passionate about it.

Reply:
Updated to 100MB per conversation.